### DISCUSSION TITLE
Compatibility with TopBand R51100A

### DISCUSSION THREAD

**Discussion by connormill** [mr-manuel/venus-os_dbus-serialbattery#70] [New BMS requests]
Hello,

I have a Topband R51100A Battery pack and may be picking up some more.

I can get the BMS to communicate just find over RS485 and their windows application using the Waveshare RS485 adapter set to 9600 baud rate

I have followed some guids on YouTube to install Venus OS on a rPi 4 and have installed the latest build of the driver but I am not getting the battery to show up

I have been looking at Dip Switch settings in the Topband manual and it does mention Victron compatibility but as yet I have had no luck.

Is it simply that the BMS in this pack is not compatibile, or am I missing somehting in config to get it reading correctly

Thanks

**Comment by atomi23** [comment]
I got also a Topband Battery Pack with a Topband BMS Battery  Model:RS-R4830A-50479-01 : BMS Topband IT4850YT-V02

I tried with Gemini (KI) to reverse the Windows BMS software and he says it looks like ACSI , but didn´t work and AI reset , i will trie some wireshare this week maybe
